examples/flava/native/train.py

```diff
@@ -165,36 +189,17 @@ def create_model(self) -> torch.nn.Module:
             model = FSDP(
                 model,
                 mixed_precision=mp,
+                device_id=self.device,
                 auto_wrap_policy=partial(
                     transformer_auto_wrap_policy,
-                    transformer_layer_cls={TransformerEncoderLayer},
+                    transformer_layer_cls={
+                        TransformerEncoderLayer,
```

Comment:
remove

Reply:
@ankitade  i thought we wanted to keep this for the imagenet validation?